setup.py

```diff
@@ -392,6 +396,15 @@ def run(self):
 with open(requirements_path) as f:
     install_requires = f.read().splitlines()
 
+
+if is_manylinux:
+    AUDITWHEEL_PLAT = environ.get('AUDITWHEEL_PLAT', None)
```

Comment:
Is this env var set by default or is it optional? Is it documented?

Reply:
It is optional. It is set by manylinux docker images when the image were built.  It is not set on Ubuntu docker images.